plutus-ledger-api/src/Plutus/V1/Ledger/Scripts.hs

```diff
@@ -316,6 +317,14 @@ instance BA.ByteArrayAccess StakeValidator where
     withByteArray =
         BA.withByteArray . BSL.toStrict . serialise
 
+-- | Script runtime representation of a @Digest SHA256@.
+newtype ScriptHash =
```

Comment:
we probably should put this into a `V2.Ledger.Scripts` module and leave V1 as it is - @michaelpj  ?

Reply:
No, this is okay. The constraint is "none of the stuff exported from `V1.Api` should change - adding new stuff here is okay. There's a bunch of stuff in here that isn't exported from `V1.Api` just because I wasn't organized enough to put it elsewhere :sweat_smile:

Reply:
@michaelpj Then, I'll leave `ScriptHash` in `Plutus.V1.Ledger.Scripts`. Should I export it in `Plutus.V2.Ledger.Api`?

Reply:
No. The stuff exported in the `Api` modules is exactly what the ledger needs and no more. They don't need this,  so don't export it.

Reply:
Alright, thanks!